src/bin_cct.cmake

```diff
@@ -14,6 +14,6 @@ target_compile_options(cct PRIVATE ${PROJ_CXX_WARN_FLAGS})
 install(TARGETS cct
   RUNTIME DESTINATION ${BINDIR})
 
-if(MSVC AND BUILD_LIBPROJ_SHARED)
+if(MSVC AND BUILD_SHARED_LIBS)
   target_compile_definitions(cct PRIVATE PROJ_MSVC_DLL_IMPORT=1)
 endif()
```

Comment:
A personal preference comment: I've learned to strive for declarative CMake scripting and avoid `if`-s. I'd probably have rewritten those `if`-s using the generator expressions:

Do this ones:
```
set(msvc_build_shared "$<AND:$<CXX_COMPILER_ID:MSVC>,$<BOOL:${BUILD_SHARED_LIBS}>>")
```

Then reuse
```
target_compile_definitions(cct PRIVATE $<${msvc_build_shared}:PROJ_MSVC_DLL_IMPORT>)
```